readme.md

```diff
@@ -64,6 +64,13 @@ Default: `'-'`
 
 Custom hostname.
 
+##### route
```

Comment:
I don't think `route` is the best name as it's more commonly used for a whole route, not just part of it.

Reply:
Makes sense – this was a carryover from a variable name in my app. What do you think of `file` or `fileName`. I wasn't sure if it made sense to also require the extension but I couldn't think of a reason why it wouldn't be an HTML file.